detector.py

```diff
@@ -426,15 +468,25 @@ def loadDetectionsFromFile(self, path):
                     distance = float(split_line[2])
                     angle = float(split_line[3])
                     aspect = float(split_line[4])
+                    l2a_ratio = float(split_line[5])
+                    frame_pc_time = int(split_line[6] * 1e6)
 
-                    c1 = [float(split_line[6]), float(split_line[5])]
-                    c2 = [float(split_line[8]), float(split_line[7])]
-                    c3 = [float(split_line[10]), float(split_line[9])]
-                    c4 = [float(split_line[12]), float(split_line[11])]
+                    c1 = [float(split_line[8]), float(split_line[7])]
+                    c2 = [float(split_line[10]), float(split_line[9])]
+                    c3 = [float(split_line[12]), float(split_line[11])]
+                    c4 = [float(split_line[14]), float(split_line[13])]
```

Comment:
Minor tip: we could access these columns by an identifier, instead of relying on hard coded numerical indexes, to ensure consistency between functions. Something like a CSV DictReader maybe?